src/cdn/cloudflare.cls.php

```diff
@@ -76,6 +76,10 @@ private function _get_devmode($show_msg = true)
 		Debug2::debug('[Cloudflare] _get_devmode result ', $res);
 
 		$curr_status = self::get_option(self::ITEM_STATUS, array());
+		// Make sure is array: #992174
+		if(!$curr_status){
+			$curr_status = array();
```

Comment:
Why not use 
`$curr_status = self::get_option(self::ITEM_STATUS, array()) ?: array();`?

Reply:
Added